src/app/core/components/navigation/component.ts

```diff
@@ -13,10 +13,17 @@
 // limitations under the License.
 
 import {Component, Input, OnDestroy, OnInit} from '@angular/core';
+import {NavigationEnd, Router} from '@angular/router';
 import {Auth} from '@core/services/auth/service';
 import {UserService} from '@core/services/user';
 import {merge, of, Subject} from 'rxjs';
-import {switchMapTo, take, takeUntil} from 'rxjs/operators';
+import {switchMapTo, take, takeUntil, filter} from 'rxjs/operators';
+
+const enum ViewName {
+  Projects = 'Projects',
+  Account = 'User Settings',
```

Comment:
IMO, better to use full names

UserSettings: "User Settings" -> Instead of `Account`
AdminSettings: "Admin Settings" -> Instead of `Settings`

this is my personal opinion but @maciaszczykm  does have a better context :)

Reply:
True, "User Settings" and "Admin Panel"/"Admin Settings" should be used. This is how these views are named in the app already.

Reply:
@KhizerRehan comments been fixed